src/components/input/StringField.jsx

```diff
@@ -1,36 +1,56 @@
+/**
+ * Use Cases:
+ * - component receives value => string | null
+ * - user types in the input => string (maybe empty)
+ * - user clicks on the reset button => null
+ *
+ * Notes: useRef might not be needed here, as long as value={value ?? ''} does
+ * not trigger an onChange event.
+ */
+
 import React, { useRef } from 'react';
-import { useTranslation } from 'react-i18next';
-import { makeStyles } from '@material-ui/core/styles';
-import TextField from '@material-ui/core/TextField';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import { TextField } from '@material-ui/core';
 
-export default function StringField({ value, ...props }) {
-  const { t } = useTranslation();
-  const computedValue = useRef(value);
-  const useStyles = makeStyles((theme) => ({
-    textField: {
-      margin: 'auto',
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      display: 'flex',
+      alignItems: 'start',
+    },
+    leftIcon: {
+      width: '2rem',
+      height: '2rem',
+      marginTop: theme.spacing(7),
+      marginRight: theme.spacing(2),
```

Comment:
Changed these to use `rem` units. There is another branch in progress where the provided Material UI theme already does this.